Translators/WeatherStations/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.10-slim
+
+ENV PYTHONUNBUFFERED=TRUE
+
+ENV WORKDIR=/app
+
+WORKDIR $WORKDIR
+
+COPY Translators/WeatherStations/ ./Translators/WeatherStations/
+
+COPY Translators/Shared/ ./Translators/Shared/
+
+ENV FLASK_APP=./Translators/WeatherStations/main.py
+
+ENV PYTHONPATH="/app:${PYTHONPATH}"
+
+RUN apt-get update && apt-get install -y libgeos-dev
```

Comment:
question: what's libgeos for?

Reply:
This was required for the geospatial libraries originally used by the WZDx translator, but isn't required for the Weather Stations translator. This has been removed.